pkg/types/baremetal/defaults/platform.go

```diff
@@ -18,8 +21,32 @@ const (
 	HardwareProfile         = "default"
 	APIVIP                  = ""
 	IngressVIP              = ""
+	ProvisioningInterface   = "ens3"
+	ProvisioningNetworkCIDR = "172.22.0.0/24"
+	ProvisioningHTTPPort    = "6180"
+	DeployKernelURL         = "http://#ProvisioningIP#:6180/images/ironic-python-agent.kernel"
+	DeployRamdiskURL        = "http://#ProvisioningIP#:6180/images/ironic-python-agent.initramfs"
+	IronicEndpoint          = "http://#ProvisioningIP#:6385/v1/"
+	IronicInspectorEndpoint = "http://#ProvisioningIP#:5050/v1/"
```

Comment:
Instead of calculating the URL/Endpoint defaults here I was thinking we'd calculate them in the BMO, since that's the only place that needs to know these values and it's an implementation detail specific to the ironic backend of the BMO?

Reply:
@sadasu We need to integrate this into your work too.